paz/models/keypoint/keypointnet.py

```diff
@@ -11,6 +11,14 @@
 from ..layers import ExpectedDepth
 from ..layers import ExpectedValue2D
 from ..layers import SubtractScalar
+from tensorflow.keras.models import Model
+from tensorflow.keras.initializers import HeNormal
+from tensorflow.keras.constraints import MaxNorm
+from tensorflow.keras.layers import (
+    ReLU,
+    Dense,
+    Dropout
+)
```

Comment:
just check if this is needed if not then remove it. Check if the imports are used.

Reply:
resolved